klaviyo/public.py

```diff
@@ -63,8 +69,22 @@ def track(
         if ip_address:
             params['ip'] = ip_address
 
-        query_string = self._build_query_string(params, is_test)
-        return self._public_request(self.TRACK, query_string)
+        method = method.lower()
```

Comment:
I believe the logic from here to the end of the method is identical with the identify method with the exception of the track vs identify string. this would be pretty easy to dry out

Reply:
updated